src/components/marker.ts

```diff
@@ -90,14 +90,26 @@ function Marker(props: MarkerProps) {
 
   const marker: MapboxMarker = useMemo(() => {
     let hasChildren = false;
+    let childElement = null;
+
     React.Children.forEach(props.children, el => {
       if (el) {
         hasChildren = true;
+
+        const element = el as JSX.Element;
+        const {children, ...other} = element.props;
+
+        childElement = document.createElement('div');
+
+        if (other['aria-label']) {
```

Comment:
This seems to be a strange way to handle aria label (what if multiple child elements have it defined?)... Is it easier to just add a new prop to the Maker component itself?